workflow_calcium_imaging/paths.py

```diff
@@ -22,16 +37,7 @@ def get_scan_image_files(scan_key):
         FileNotFoundError: If the session directory or tiff files are not found.
     """
     # Folder structure: root / subject / session / .tif (raw)
-    data_dir = get_imaging_root_data_dir()
-
-    from .pipeline import session
-
-    sess_dir = data_dir / (session.SessionDirectory & scan_key).fetch1("session_dir")
-
-    if not sess_dir.exists():
-        raise FileNotFoundError(f"Session directory not found ({sess_dir})")
-
-    tiff_filepaths = [fp.as_posix() for fp in sess_dir.glob("*.tif")]
+    sess_dir, tiff_filepaths = _find_files_by_type(scan_key, "*.tif")
```

Comment:
A tif file is not always a ScanImage recording but a ScanImage recording is a tif file.

Reply:
@tdincer Is there a fix you can recommend? I believe my changes here are functionally equivalent to the previous code
```python
tiff_filepaths = [fp.as_posix() for fp in sess_dir.glob("*.tif")]
```
My changes extract this pattern to a common helper function

Reply:
Your changes are fine. I'm just writing one idea but we don't have to apply it now. If the file is a ScanImage file and needs to be read by scanreader, the scanner information can be verified from its header. But reading the header introduces an overhead.